---
float32[6] curangle
